Windows 10 installation ERROR: Exception calling "RemoveAccessRule"

"choco install osquery" command fail with this error message on my Windows 10 Family PC.

ERROR: Exception calling "RemoveAccessRule" with "1" argument(s): "Some or all identity references could not be translated."
The install of osquery was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\osquery\tools\chocolateyinstall.ps1'.

Test from admin account with admin console shell on a French Windows 10 Family version 1607 (14393.1066)

Full message in log file
2017-04-28 15:31:38,203 [INFO ] - VERBOSE: Importing alias 'Start-ChocolateyProcess'.
2017-04-28 15:31:38,375 [ERROR] - ERROR: Exception calling "RemoveAccessRule" with "1" argument(s): "Some or all identity references could not be translated."
2017-04-28 15:31:38,781 [ERROR] - The install of osquery was NOT successful.
2017-04-28 15:31:38,797 [ERROR] - Error while running 'C:\ProgramData\chocolatey\lib\osquery\tools\chocolateyinstall.ps1'.
 See log for details.
2017-04-28 15:31:40,875 [WARN ] - 
Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
2017-04-28 15:31:40,891 [INFO ] - 
2017-04-28 15:31:40,907 [ERROR] - Failures
2017-04-28 15:31:40,922 [ERROR] -  - osquery (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\osquery\tools\chocolateyinstall.ps1'.
 See log for details.
2017-04-28 15:31:40,938 [WARN ] -

--- a/tools/provision/chocolatey/osquery_utils.ps1
+++ b/tools/provision/chocolatey/osquery_utils.ps1
@@ -24,7 +24,9 @@ function Set-DenyWriteAcl {
     $propagationFlag = [System.Security.AccessControl.PropagationFlags]::None
     $permType = [System.Security.AccessControl.AccessControlType]::Deny
 
-    $permission = "everyone", "write", $inheritanceFlag, $propagationFlag, $permType
+    $worldSIDObj = New-Object System.Security.Principal.SecurityIdentifier ('S-1-1-0')
+    $worldUser = $worldSIDObj.Translate( [System.Security.Principal.NTAccount])
+    $permission = $worldUser.Value, "write", $inheritanceFlag, $propagationFlag, $permType
     $accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule $permission
     # We only support adding or removing the ACL
     if ($action -ieq 'add') {

